src/scss/vendors/_vendors.scss

```diff
@@ -1 +1,60 @@
 // Vendor-specific styles.
+
+// UIO
+
+// uio tab
+#defaultContainer .fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
+	background: $white;
+	border: rem(1) solid $green-dark;
+	border-radius: 0 0 rem(30) rem(30);
+	border-top: none;
+	box-shadow: none;
+	box-sizing: border-box;
+	padding: rem(10) rem(20);
+
+	button {
+		color: $black;
+		font-family: "Roboto", sans-serif;
+		font-size: rem(16);
+		font-style: normal;
+		font-weight: $font-weight-normal;
+		line-height: rem(19);
+	}
+
+	&:hover,
+	&:focus-within {
+		border: rem(3) solid $green-dark;
+		border-top: none;
+		margin-right: rem(70);
+
+		button {
+			font-weight: $font-weight-bold;
+		}
+	}
+}
+
+// focus
+a[href="https://www.ic.gc.ca/eic/site/118.nsf/eng/home"],
+a[href="https://idrc.ocadu.ca/"],
+a[href="https://ocadu.ca/"] {
+	padding: rem(5);
+}
+
+#defaultContainer .content-wrapper .pagination .pagination-list {
+	.pagination-next,
+	.pagination-previous {
+		box-shadow: none;
+	}
+}
+
+@each $theme, $foregroundColor, $backgroundColor in $uio-themes {
+	.fl-theme-#{$theme} {
```

Comment:
Let's take the chance to apply UIO specific styles to UIO+ themes too, which uses the css selector in the format of `.gpii-ext-theme-#{$theme}`.